### PR TITLE
Controller for student fees

### PR DESCRIPTION
**Pull request by CobbyElsonfx**

### CODE REVIEW

File: CorsaSchoolManagementSystem/Views/studentDetails.php [hunk 1]
```diff
@@ -370,7 +370,7 @@
                   <input type="text" name="status" class="form-control">
                 </div>
                 <div class="container mt-3 mb-2">
-                  <input type="submit" name="payFee" class="form-control btn btn-sm btn-primary fw-bold py-2">
+                  <input type="addfees" name="payFee" class="form-control btn btn-sm btn-primary fw-bold py-2">
```
Comment (OliverMensahDev) [inline CorsaSchoolManagementSystem/Views/studentDetails.php]: no need to change this. All you need to listen is the name attribute on your submission.

File: CorsaSchoolManagementSystem/studentFees.php [hunk 2]
```diff
@@ -0,0 +1,29 @@
+<?php 
+
+require('Core/init.php');
+
+if($_SESSION['loggedIn'] && $_SESSION['userType'] === UserType::ADMIN){
+    if($_POST['addfees']){
+        //student fees
+        $admin = new Admin(new Database());
+        $studentFees = new studentFees();
```
Comment (OliverMensahDev) [inline CorsaSchoolManagementSystem/studentFees.php]: Where from the studentFees class? Was it created earlier, if not you need to create it?

File: CorsaSchoolManagementSystem/studentFees.php [hunk 3]
```diff
@@ -0,0 +1,29 @@
+<?php 
+
+require('Core/init.php');
+
+if($_SESSION['loggedIn'] && $_SESSION['userType'] === UserType::ADMIN){
+    if($_POST['addfees']){
+        //student fees
+        $admin = new Admin(new Database());
+        $studentFees = new StudentFees();
```
Comment (OliverMensahDev) [inline CorsaSchoolManagementSystem/studentFees.php:9]: Where is the class file for `StudentFees`?

File: CorsaSchoolManagementSystem/Views/studentDetails.php (hunk 1 above)
Comment (CobbyElsonfx) [reply CorsaSchoolManagementSystem/Views/studentDetails.php]: Alright please

File: CorsaSchoolManagementSystem/Models/StudentFees.php [hunk 4]
```diff
@@ -0,0 +1,16 @@
+<?php 
+
+require_once("Core/init.php");
```
Comment (OliverMensahDev) [inline CorsaSchoolManagementSystem/Models/StudentFees.php]: How does the StudentFees Class depend on the init file?

Comment (CobbyElsonfx) [reply CorsaSchoolManagementSystem/Models/StudentFees.php]: The init file contains PHP error configurations which displays php errors if there is.

Comment (OliverMensahDev) [reply CorsaSchoolManagementSystem/Models/StudentFees.php]: This class won't produce any error or exception which requires that file. The only error likely to have would be syntax error and the syntax is correct so would be recommended to remove that line of code.

Comment (OliverMensahDev) [reply CorsaSchoolManagementSystem/Models/StudentFees.php]: Also make sure if you resolve any issue mentioned via comment by clicking on resolve conversation button

Comment (CobbyElsonfx) [reply CorsaSchoolManagementSystem/Models/StudentFees.php]: Aright pls

**Review by OliverMensahDev — APPROVED** [review]

**Pull request merged**